src/components/link/link.hbs

```diff
@@ -1,9 +1,10 @@
-<!-- 
+<!--
   Copyright IBM Corp. 2016, 2018
 
   This source code is licensed under the Apache-2.0 license found in the
   LICENSE file in the root directory of this source tree.
 -->
 
 <a href="#" class="{{@root.prefix}}--link">Link</a>
-<a href="#" class="{{@root.prefix}}--link" tabindex="-1" aria-disabled="true">Link</a>
+<a class="{{@root.prefix}}--link" tabindex="-1" aria-disabled="true">Link</a>
```

Comment:
Don't think we need `aria-disabled="true"` on the active one

Reply:
it's not actually active, it's supposed to be a disabled "link" with no `href`. But I was making the case for not supporting this since an anchor without an `href` is pretty much just text